src/routes/search.py

```diff
@@ -25,27 +26,31 @@ def get_blueprint():
 
 
 @search_blueprint.route("/search_ingredients", methods=["POST"])
-def search_ingredients():
-    if flask.request.method == "POST":
-        ingredients = flask.request.form.get("searchIngredients")
+def searchByingredients():
+    if request.method == "POST":
+        ingredients = request.form.get("searchIngredients")
         # prints what the user searches for to the terminal
         print(ingredients)
-        returnedDict = api.spoonacular.search_ingredients(ingredients)
+        returnedDict = search_ingredients(ingredients)
         # prints the 10 recipes to the terminal
         print(returnedDict)
+        if returnedDict == []:
+            print("error")
```

Comment:
I suggest that once the search function is done, you should handle the error by displaying a message being sent to the HTML during the render_template call instead of just printing error.